packages/icon/src/vaadin-icon-mixin.js

```diff
@@ -173,20 +171,6 @@ export const IconMixin = (superClass) =>
       this.__fetch = fetch.bind(window);
     }
 
-    /** @protected */
-    get slotStyles() {
-      const tag = this.localName;
-      return [
-        `
-        ${tag}[icon-class] {
-          display: inline-flex;
-          vertical-align: middle;
-          font-size: inherit;
```

Comment:
Removing this causes the test failures – in the test the `.my-icon-font` class sets `font-size: 24px` and `line-height: 1.5` which results in a `24px * 1.5 = 36px` size icon. But this class overrides the custom font icon class.

I wonder what the reason for these slotted styles was in the first place?

Reply:
Here's the relevant PR: https://github.com/vaadin/web-components/pull/6418

> `<vaadin-icon>` currently has width: `var(--lumo-icon-size-m);` as the [default in Lumo styles](https://github.com/vaadin/web-components/blob/cdab2a57b38092541ed946e0b6fe610e0ce14b9b/packages/icon/theme/lumo/vaadin-icon-styles.js#L8-L9).

> Because of this, using a custom font size directly on `<vaadin-icon>` might have an unexpected effect on the component's size. For example, the [default use of Material icons](https://developers.google.com/fonts/docs/material_icons) would apply font-size: 24px to the <vaadin-icon> element which would effectively make the icon have 36px size (= 24px * 1.5 (--lumo-icon-size-m)).

Reply:
Thanks. Okay, so we _don't_ want font-size to be used to size the icons, ever? I did have `:host { font-size: inherit !important; }` for a moment, but then started wondering if sizing font icons with font size was actually desirable. 

But perhaps not, and we could add that back in the shadow root. Or was the reasoning with the slotted styles so that we’d override the Material Icons case specifically, but still allow more specific selectors to override the font size?

Reply:
Added `font-size: inherit !important`.